src/python/foglamp/translators/omf_translator.py

```diff
@@ -936,6 +941,13 @@ def omf_object_creation():
         _log.error(message)
         raise Exception(message)
 
+async def _update_statistics():
+    global _num_readings
+    global _num_discarded_readings
+    await statistics.update_statistics_value('SENT', _num_sent)
+    _num_readings = 0
+    await statistics.update_statistics_value('UNSENT', _num_unsent)
```

Comment:
Where are we computing _num_unsent value? Shouldn't it be inside the exception of the method send_omf_message_to_end_point?